developers/documentation/how_to_submit_pr_online.md

```diff
@@ -0,0 +1,111 @@
+---
+jupytext:
+  formats: ipynb,md:myst
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.10.3
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
+---
+
+# How to open a pull request on GitHub for new documentation
+
+This how-to will guide you to open a pull request in GitHub to contribute your prepared document to napari.org. 
+
+This guide is strictly designed for adding new documents - if you want to contribute code to napari check out [this guide instead](../contributing.md).
+
+```{note}
+Don't yet a have a document ready? Check out our [main guide](./index.md) to find templates and learn how to prepare your document!
+```
+
+## Prerequisites
+
+- a new document for napari.org
+    - Your document should already be prepared for submission. Check out [our contributor's guide](./index.md) to learn how to prepare your document
+- a [GitHub account](https://github.com)
+
+## 1. Fork the napari repository
```

Comment:
Honestly, I suspect the best first step is to ask people to open an issue to suggest the new documentation/tutorial they're thinking of adding, to see if there's enthusiasm before putting in a lot of work.
Do you have thoughts, @sofroniewn & @jni?

Reply:
Yeah we could have something like `0. Before you start`. I don't think making an issue is required, but I do think it should be encouraged, especially if more than a small contribution. Typos, clear fixes etc are fine without issues. Adding something to that effect might help

Reply:
Thanks for the feedback @GenevieveBuckley and @sofroniewn - I've added a 0th step guiding users to open an issue if they'd like to add a new document